x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ILegacyClusterClient, Logger } from '../../../../../src/core/server';
+import { AuthenticationProvider } from '../../common/types';
+import { ConfigType } from '../config';
+
+export interface SessionIndexOptions {
+  readonly clusterClient: ILegacyClusterClient;
+  readonly tenant: string;
+  readonly config: Pick<ConfigType, 'session'>;
+  readonly logger: Logger;
+}
+
+/**
+ * Version of the current session index template.
+ */
+const SESSION_INDEX_TEMPLATE_VERSION = 1;
+
+/**
+ * Alias of the Elasticsearch index that is used to store user session information.
+ */
+export const SESSION_INDEX_ALIAS = '.kibana_security_session';
```

Comment:
I'm torn on this too. Although the chance of this collision is small I agree this may happen. The second option sounds better to me since it'd simplify our code as well and we may easily return back to the hardcoded value if we ever decide do drop `kibana.index` config option. But if we introduce a new config option just for that case we'll have to support it in the future.

Let me try, thanks!